### PR TITLE
frontend/ambulance rc fix

### PR DESCRIPTION
**Pull request by Kuldeepkumawat144**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs [hunk 1]
```diff
@@ -1928,7 +1928,7 @@ addAadhaarNumber push state visibility' =
   , stroke $ "1," <> Color.black600
   , padding $ Padding 20 16 20 16
   , gravity CENTER_VERTICAL
-  , onClick push $ const ClickAddAlternateButton
+  , onClick push $ const LinkAadhaarAC
```
Comment (slow-codex) [inline Frontend/ui-driver/src/Screens/HomeScreen/View.purs:1931]: We can remove `ClickAddAlternateButton` from Controller if it's not used anywhere else.

File: Frontend/ui-driver/src/Flow.purs [hunk 2]
```diff
@@ -150,6 +150,7 @@ import Effect.Unsafe (unsafePerformEffect)
 import Common.Types.App as CTA
 import AssetsProvider (renewFile)
 import Control.Bind
+import Data.Array (elem)
```
Comment (ashishsingh101) [inline Frontend/ui-driver/src/Flow.purs:153]: not required

File: Frontend/ui-driver/src/Flow.purs [hunk 3]
```diff
@@ -1253,13 +1254,18 @@ driverProfileFlow = do
       case res of
         Right (MakeRcActiveOrInactiveResp response) -> do
           pure $ toast $ if state.data.isRCActive then "RC-"<>state.data.rcNumber<>" "<> (getString DEACTIVATED) else "RC-"<>state.data.rcNumber<> (getString IS_ACTIVE_NOW)
+          globalstate <- getState
           if state.data.isRCActive then do
-            globalstate <- getState
             (GetDriverInfoResp getDriverInfoResp) <- getDriverInfoDataFromCache globalstate false
             let status = getDriverStatus $ fromMaybe "" getDriverInfoResp.mode
+            _ <- pure $ spy "debug rc" status
```
Comment (ashishsingh101) [inline Frontend/ui-driver/src/Flow.purs:1261]: remove spy

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs (hunk 1 above)
Comment (Kuldeepkumawat144) [reply Frontend/ui-driver/src/Screens/HomeScreen/View.purs:1931]: Removed

**Review by ashishsingh101 — APPROVED** [review]

**Pull request merged**